src/analysis/complex/circle.lean

```diff
@@ -97,3 +99,76 @@ def exp_map_circle_hom : ℝ →+ (additive circle) :=
   map_zero' := by { rw exp_map_circle, convert of_mul_one, simp },
   map_add' := λ x y, show exp_map_circle (x + y) = (exp_map_circle x) * (exp_map_circle y),
     from subtype.ext $ by simp [exp_map_circle, exp_add, add_mul] }
+
+lemma exp_map_circle_eq_one_iff {x : ℝ} :
+  exp_map_circle x = 1 ↔ x ∈ add_subgroup.gmultiples (2 * π) :=
```

Comment:
I wonder if it makes sense to express this (either in place of this lemma or additionally) as `exp_map_circle_hom.ker = add_subgroup.gmultiples (2 * π)`?